### PR TITLE
Add Milestones and ReleasedAt

### LINKED ISSUE #991: Add Milestones and ReleasedAt in CreateReleaseOptions and UpdateReleaseOptions
Threre are `milestones` and `released_at` in the document of create-a-release and update-a-release.   
But, CreateReleaseOptions and UpdateReleaseOptions dose'nt has them.
- https://docs.gitlab.com/ce/api/releases/index.html#create-a-release
- https://docs.gitlab.com/ce/api/releases/index.html#update-a-release

### PR DESCRIPTION
**Pull request by WinterYukky**
Resolves #991

- [x] add Milestones and ReleasedAt to CreateReleaseOptions
- [x] add Milestones and ReleasedAt to UpdateReleaseOptions
- [x] add unit test.

### CODE REVIEW

File: releases.go [hunk 1]
```diff
@@ -124,7 +124,9 @@ type CreateReleaseOptions struct {
 	TagName     *string        `url:"tag_name" json:"tag_name"`
 	Description *string        `url:"description" json:"description"`
 	Ref         *string        `url:"ref,omitempty" json:"ref,omitempty"`
+	Milestones  *[]string      `url:"milestones,omitempty" json:"milestones,omitempty"`
```
Comment (svanharmelen) [inline releases.go:127]: This should not be a pointer as the zero value of a slice is already `nil`

File: releases.go [hunk 2]
```diff
@@ -157,8 +159,10 @@ func (s *ReleasesService) CreateRelease(pid interface{}, opts *CreateReleaseOpti
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/releases/index.html#update-a-release
 type UpdateReleaseOptions struct {
-	Name        *string `url:"name" json:"name"`
-	Description *string `url:"description" json:"description"`
+	Name        *string    `url:"name" json:"name"`
+	Description *string    `url:"description" json:"description"`
+	Milestones  *[]string  `url:"milestones,omitempty" json:"milestones,omitempty"`
```
Comment (svanharmelen) [inline releases.go:164]: Same here about the pointer...

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Looks good, but needs a small tweak...

**Comment by WinterYukky** [comment]
@svanharmelen Thank you for quickly review😊
I fixed it.  Please check.

**Review by svanharmelen — APPROVED** [review]

**Comment by svanharmelen** [comment]
Thanks @WinterYukky! LGTM!

**Pull request merged**